kroxylicious-systemtests/pom.xml

```diff
@@ -102,7 +102,8 @@
         </dependency>
         <dependency>
             <groupId>io.kroxylicious</groupId>
-            <artifactId>kroxylicious-api</artifactId>
+            <artifactId>kroxylicious-record-encryption</artifactId>
```

Comment:
the system tests are running against the container image which has the filters, so this dependency should not be needed.

Reply:
I used it to easy the creation of the config map of Kroxylicious, I would like to remove it because it is causing me a headache